src/main/webapp/js/in_shift.js

```diff
@@ -50,14 +45,18 @@ function clockout(){
 	var url = SCHEDULER_APP.url_clockin + '/clockout';
 	//get params from page
 	var params = {
-			employeeId: 1,
+			employeeId: 6,
```

Comment:
can we change this now to use the employee that's logged in, maybe in a different PR